File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.*;

public class Main {
    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    static StringTokenizer st;
    static StringBuilder sb = new StringBuilder();

    public static void main(String[] args) throws IOException {

        int A = Integer.parseInt(br.readLine());
        int B = Integer.parseInt(br.readLine());
        int C = Integer.parseInt(br.readLine());

        if (A == 60 && B == 60 && C == 60) {
            System.out.println("Equilateral");
        } else if (A + B + C == 180 && (A == B || B == C || C == A)) {
            System.out.println("Isosceles");
        } else if (A + B + C == 180 && A != B && B != C) {
            System.out.println("Scalene");
        } else if (A + B + C != 180) {
            System.out.println("Error");
        }


    } // end of main

} // end of class

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.py
"""
"""
import sys

input = lambda: sys.stdin.readline().strip()

lst = []
result = 0
for _ in range(3):
    lst.append(int(input()))

if max(lst) == 60 and min(lst) == 60:
    print("Equilateral")
elif sum(lst) == 180 and (lst[0] == lst[1] or lst[1] == lst[2] or lst[2] == lst[0]):
    print("Isosceles")
elif sum(lst) == 180 and (lst[0] != lst[1] and lst[1] != lst[2]):
    print("Scalene")
elif sum(lst) != 180:
    print("Error")
